source/Engine.h

```diff
@@ -102,6 +102,8 @@ class Engine {
 
 
 private:
+	using BufferIndex = bool;
```

Comment:
I'm still in favor of dropping this ~~it's unnecessary~~

Reply:
okay >:(